### DISCUSSION TITLE
Error: Cannot find module 'webpack-dev-server' for running 'webpack serve'.

### DISCUSSION THREAD

**Discussion by JohnAdam-123** [nasa/openmct#7921] [Community Q&A] [answered]
I am trying to running openMCT-yamcs project in Virtual Machine but whenever I run `npm install`  or `npm start` I get error as an output that webpack module is not found. 
![npm start error](https://github.com/user-attachments/assets/90f23156-b262-4d20-9b42-88d4b3fc2814)

How can I solve this, any ideas please?

**Comment by shefalijoshi** [comment, marked as answer]
Hi @JohnAdam-123 ,

We need a bit more information to debug this further:
1. Which virtual machine are you using and does it have access to the internet to download npm packages?
2. Which version/branch of openmct-yamcs are you trying to install? Is it master?
3. What version of node are you using? Is it greater than node 18.14.2  and less than node 22?
4. At which step of the [README](https://github.com/akhenry/openmct-yamcs/blob/master/README.md#installation) are you getting the errors above?

**Reply by JohnAdam-123** [reply]
Ooh... thank you for the concern, am no longer dealing with this project. It was resolved already if am not mistaken. I appreciate the support.
